tests/load-tests.php

```diff
@@ -331,6 +301,6 @@ static function () {
 		WP_Filesystem();
 
 		// Simulate that the original object-cache.copy.php file exists.
-		$wp_filesystem->put_contents( PERFLAB_PLUGIN_DIR_PATH . 'server-timing/object-cache.copy.php', file_get_contents( PERFLAB_PLUGIN_DIR_PATH . 'server-timing/object-cache.copy.php' ) );
+		$wp_filesystem->put_contents( PERFLAB_PLUGIN_DIR_PATH . 'includes/server-timing/object-cache.copy.php', file_get_contents( PERFLAB_PLUGIN_DIR_PATH . 'includes/server-timing/object-cache.copy.php' ) );
```

Comment:
This looks good, but there's still another production reference to this file that you need to update: Please check the main PL `load.php` file. It still references the old location, causing a unit test to fail.

Reply:
I have fixed the path for the server-timing API. 

@felixarntz I am not sure of all the testing steps for this API. Would you please check if it's working on your local? Else please share the steps and I'll test it as per requirements.

Reply:
@thelovekesh The logic places this file to have parts of the Server-Timing API eligible to load sooner. The extra checks for placing the file though are needed since some sites may already have an `object-cache.php` drop-in present, or the server environment may prevent writing that file. So all of this has to be handled gracefully.